src/Parser.php

```diff
@@ -2822,19 +2822,37 @@ private function parseScopedPropertyAccessExpression($expression, $fallbackParen
         return $scopedPropertyAccessExpression;
     }
 
+    public function isParsingObjectCreationExpression($expression) {
```

Comment:
Is it typical in PHP to have a method and property with the same names? It threw me off.